src/test/Testcafe/page-objects/designerPage.js

```diff
@@ -16,5 +13,28 @@ export default class designerPage {
     this.fileUpload = Selector(".button").withText("FileUpload");
     this.submit = Selector(".button").withText("Submit");
     this.addContainer = Selector(".button").withText("Add Container");
+    this.saveButton = Selector(".button").withText("Save");
+    this.addApiConnection = Selector(".d-block").withText("Api connections").child(0);
+    this.addRuleConnection = Selector(".d-block").withText("Rule connections").child(0);
+    this.addConditionalRendering = Selector(".d-block").withText("Conditional Rendering").child(0);
+    //this
+
+    //preview tab
+    this.previewSaveButton = Selector(".a-btn-success").withText("Save");
+    this.controlAndSubmit = Selector(".a-btn-success");
+
+    //tekst tab
+    this.nyTextButton = Selector(".btn").withText("Ny tekst");
+    this.addNewLanguage = Selector("#newtab > a");
+  }
+
+
+  deleteDataModelTexts(numberToDelete) {
+    var deleteButtons = await Selector(".tbn").withText("Slett");
+    var count = deleteButtons().count;
+
+    for (var i = 0; i < numberToDelete && i < count; i++) {
```

Comment:
use `let` instead of `var`
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/let

Reply:
Fixed